src/panic-implementation.md

```diff
@@ -0,0 +1,35 @@
+## #[panic_implementation]
+
+The `#[panic_implementation]` attribute can only be applied to a function with signature
+`fn(&PanicInfo) -> !`. The function marked with this attribute defines the behavior of `panic!` in
+`#![no_std]` applications. There must be a *single* `#[panic_implementation]` function in the
+dependency graph of a binary / dylib / cdylib crate.
+
+The `PanicInfo` struct contains information about the location of the panic. The API of `PanicInfo`
+can be found in the [API docs]. As of 1.28-beta (2018-06-21) there's a difference between
```

Comment:
We don't discuss anything that's not stable. This includes beta versions. Nor do we discuss older versions.